rbi/rootfs-img/image/build-img.sh

```diff
@@ -88,7 +89,7 @@ main() {
     local abs_source_code_dir=$(cd "$source_code_dir";pwd)
     local abs_output_dir=$(cd "$output_dir";pwd)
     local abs_rootfs_dir=$(cd "$rootfs_dir";pwd)
-    abs_pwd=$(pwd)
+    
```

Comment:
delete the empty line please

Reply:
I have rechecked all the shells and delete 3 empty lines.

Reply:
https://github.com/alibaba/inclavare-containers/pull/1255/commits/2ac58355fc80d4fc109ab6fe8b66b3f388dbb973